--- /dev/null	2024-06-03 10:08:45
+++ BUILD	2024-06-03 10:08:35
@@ -0,0 +1,15 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
+    name = "tdb",
+    lib_source = ":srcs",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@rules_python//python/cc:current_py_cc_headers",
+    ]
+)
