--- /dev/null	2024-06-22 10:41:34
+++ BUILD	2024-06-22 10:41:32
@@ -0,0 +1,21 @@
+
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "meson")
+
+filegroup(
+    name = "srcs",
+    srcs = glob(
+        ["**"],
+    ),
+)
+
+meson(
+    name = "libv4l",
+    copts = [
+        "-fPIC",
+    ],
+    lib_source = ":srcs",
+    out_shared_libs = [
+        "libpulse-simple.so",
+    ],
+    visibility = ["//visibility:public"],
+)
