diff --git a/foreign_cc/repositories.bzl b/foreign_cc/repositories.bzl
index 3350993..590523b 100644
--- a/foreign_cc/repositories.bzl
+++ b/foreign_cc/repositories.bzl
@@ -12,7 +12,7 @@ def rules_foreign_cc_dependencies(
         cmake_version = "3.23.2",
         make_version = "4.4.1",
         ninja_version = "1.12.0",
-        meson_version = "1.1.1",
+        meson_version = "1.4.0",
         pkgconfig_version = "0.29.2",
         register_preinstalled_tools = True,
         register_built_tools = True,
diff --git a/toolchains/built_toolchains.bzl b/toolchains/built_toolchains.bzl
index 7f1ecbb..3c1b3d0 100644
--- a/toolchains/built_toolchains.bzl
+++ b/toolchains/built_toolchains.bzl
@@ -195,16 +195,15 @@ def _meson_toolchain(version, register_toolchains):
         native.register_toolchains(
             "@rules_foreign_cc//toolchains:built_meson_toolchain",
         )
-    if version == "1.1.1":
+    if version == "1.4.0":
         maybe(
             http_archive,
             name = "meson_src",
             build_file_content = _MESON_BUILD_FILE_CONTENT,
-            sha256 = "d04b541f97ca439fb82fab7d0d480988be4bd4e62563a5ca35fadb5400727b1c",
-            strip_prefix = "meson-1.1.1",
+            sha256 = "8fd6630c25c27f1489a8a0392b311a60481a3c161aa699b330e25935b750138d",
+            strip_prefix = "meson-1.4.0",
             urls = [
-                "https://mirror.bazel.build/github.com/mesonbuild/meson/releases/download/1.1.1/meson-1.1.1.tar.gz",
-                "https://github.com/mesonbuild/meson/releases/download/1.1.1/meson-1.1.1.tar.gz",
+                "https://github.com/mesonbuild/meson/releases/download/1.4.0/meson-1.4.0.tar.gz",
             ],
         )
         return
