--- /dev/null	2024-06-26 12:51:08
+++ MODULE.bazel	2024-06-26 12:49:17
@@ -0,0 +1,24 @@
+module(
+    name = "onnxruntime_extensions",
+    version = "0.11.0",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.5.0",
+)
+
+bazel_dep(name = "platforms", version = "0.0.8")
+
+bazel_dep(
+    name = "rules_foreign_cc",
+    version = "0.10.1",
+)
+bazel_dep(name = "nlohmann_json", version = "3.11.3")
+bazel_dep(name = "gsl", version = "4.0.0")
+bazel_dep(name = "re2", version = "2024-06-01", repo_name = "com_googlesource_code_re2")
+
+ONNX_VERSION = "1.18.0"
+bazel_dep(name = "onnxruntime_osx_arm64", version = ONNX_VERSION, repo_name = "onnxruntime_osx_arm64")
+bazel_dep(name = "onnxruntime_linux_x64", version = ONNX_VERSION, repo_name = "onnxruntime_linux_x64")
+bazel_dep(name = "opencv", version = "3.4.20", repo_name = "opencv")
