--- CMakeLists.txt.old	2024-06-25 15:10:18
+++ CMakeLists.txt	2024-06-25 21:27:09
@@ -282,7 +282,7 @@
 # set default MSVC warning level to 3 for external dependencies
 set_msvc_c_cpp_compiler_warning_level(3)
 include(ext_ortlib)
-include(gsl)
+# include(gsl)
 
 macro(standardize_output_folder bin_target)
   set_target_properties(${bin_target} PROPERTIES
@@ -303,7 +303,7 @@
   if(NOT TARGET re2::re2)
     set(RE2_BUILD_TESTING OFF CACHE INTERNAL "")
     message(STATUS "Fetch googlere2")
-    include(googlere2)
+    # include(googlere2)
   endif()
 
   if(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
@@ -369,7 +369,7 @@
 if(OCOS_ENABLE_CV2 OR OCOS_ENABLE_VISION)
   set(_ENABLE_OPENCV ON)
   message(STATUS "Fetch opencv")
-  include(opencv)
+  # include(opencv)
 endif()
 
 if(OCOS_ENABLE_CV2)
@@ -520,7 +520,7 @@
 
 if(OCOS_ENABLE_GPT2_TOKENIZER OR OCOS_ENABLE_WORDPIECE_TOKENIZER)
   message(STATUS "Fetch json")
-  include(json)
+  # include(json)
 endif()
 
 if(_HAS_TOKENIZER)
@@ -669,8 +669,8 @@
 endif()
 
 if(OCOS_ENABLE_GPT2_TOKENIZER OR OCOS_ENABLE_WORDPIECE_TOKENIZER)
-  target_include_directories(ocos_operators PRIVATE ${nlohmann_json_SOURCE_DIR}/single_include)
-  list(APPEND ocos_libraries nlohmann_json::nlohmann_json)
+  target_include_directories(ocos_operators PRIVATE ${nlohmann_json_SOURCE_DIR})
+  # list(APPEND ocos_libraries nlohmann_json::nlohmann_json)
 endif()
 
 # If building a shared library we can't throw an internal exception type across the library boundary as the type
@@ -685,7 +685,7 @@
 endif()
 
 target_include_directories(noexcep_operators PUBLIC ${GSL_INCLUDE_DIR})
-list(APPEND ocos_libraries Microsoft.GSL::GSL)
+# list(APPEND ocos_libraries Microsoft.GSL::GSL)
 
 list(REMOVE_DUPLICATES OCOS_COMPILE_DEFINITIONS)
 target_compile_definitions(noexcep_operators PRIVATE ${OCOS_COMPILE_DEFINITIONS})
