--- /dev/null	2024-05-27 14:42:56
+++ BUILD	2024-05-27 14:42:48
@@ -0,0 +1,79 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "libxcb_srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
+    name = "libxcb",
+    lib_source = ":libxcb_srcs",
+    env = select({
+        "@platforms//os:macos": {
+            "AR": "",
+        },
+        "//conditions:default": {
+        },
+    }),
+    deps = [
+        "@xcb_proto",
+        "@libxau"
+    ],
+    out_shared_libs = [
+        "libxcb.so",
+        "libxcb.so.1",
+        "libxcb.so.1.1.0",
+        "libxcb-composite.so",
+        "libxcb-damage.so",
+        "libxcb-dbe.so",
+        "libxcb-dpms.so",
+        "libxcb-dri2.so",
+        "libxcb-dri3.so",
+        "libxcb-glx.so",
+        "libxcb-present.so",
+        "libxcb-randr.so",
+        "libxcb-record.so",
+        "libxcb-render.so",
+        "libxcb-res.so",
+        "libxcb-screensaver.so",
+        "libxcb-shape.so",
+        "libxcb-shm.so",
+        "libxcb-sync.so",
+        "libxcb-xf86dri.so",
+        "libxcb-xfixes.so",
+        "libxcb-xinerama.so",
+        "libxcb-xinput.so",
+        "libxcb-xkb.so",
+        "libxcb-xtest.so",
+        "libxcb-xv.so",
+        "libxcb-xvmc.so",
+    ],
+    out_static_libs = [
+        "libxcb.a",
+        "libxcb-composite.a",
+        "libxcb-damage.a",
+        "libxcb-dbe.a",
+        "libxcb-dpms.a",
+        "libxcb-dri2.a",
+        "libxcb-dri3.a",
+        "libxcb-glx.a",
+        "libxcb-present.a",
+        "libxcb-randr.a",
+        "libxcb-record.a",
+        "libxcb-render.a",
+        "libxcb-res.a",
+        "libxcb-screensaver.a",
+        "libxcb-shape.a",
+        "libxcb-shm.a",
+        "libxcb-sync.a",
+        "libxcb-xf86dri.a",
+        "libxcb-xfixes.a",
+        "libxcb-xinerama.a",
+        "libxcb-xinput.a",
+        "libxcb-xkb.a",
+        "libxcb-xtest.a",
+        "libxcb-xv.a",
+        "libxcb-xvmc.a",
+    ],
+    visibility = ["//visibility:public"],
+)
