--- src/lib/flags.cc.old	2024-07-26 15:59:04
+++ src/lib/flags.cc	2024-07-26 16:03:49
@@ -21,8 +21,10 @@
 static const char *private_tmpdir = getenv("TMPDIR");
 
 DEFINE_int32(v, 0, "verbosity level");
+#ifndef OPENFST_HAS_ABSL
 DEFINE_bool(help, false, "show usage information");
 DEFINE_bool(helpshort, false, "show brief usage information");
+#endif  // OPENFST_HAS_ABSL
 DEFINE_string(tmpdir, private_tmpdir ? private_tmpdir : "/tmp",
               "temporary directory");
 
@@ -79,6 +81,7 @@
     }
     *argc -= index - 1;
   }
+#ifndef OPENFST_HAS_ABSL
   if (FLAGS_help) {
     ShowUsage(true);
     exit(1);
@@ -87,6 +90,7 @@
     ShowUsage(false);
     exit(1);
   }
+#endif
 }
 
 // If flag is defined in file 'src' and 'in_src' true or is not
