--- /dev/null	2024-06-03 10:55:11
+++ BUILD	2024-06-03 10:55:06
@@ -0,0 +1,23 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+filegroup(
+    name = "srcs",
+    srcs = glob([
+        "**",
+    ]),
+)
+
+cmake(
+    name = "check",
+    build_args = [
+        "-j8",
+    ],
+    generate_args = [
+        "-GNinja",
+    ],
+    out_static_libs = [
+        "libcheck.a",
+    ],
+    lib_source = ":srcs",
+    visibility = ["//visibility:public"],
+)
