--- /dev/null	2024-06-03 11:29:06
+++ BUILD	2024-06-03 11:28:56
@@ -0,0 +1,31 @@
+
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "meson")
+
+filegroup(
+    name = "pulseaudio_srcs",
+    srcs = glob(
+        ["**"],
+        exclude = ["src/modules/alsa/mixer/samples/USB Device 0x46d:0x9a4--USB Mixer"]
+    ),
+)
+
+meson(
+    name = "libpulse",
+    options = {
+        "doxygen": "false",
+        "man": "false",
+    },
+    copts = [
+        "-fPIC",
+    ],
+    lib_source = ":pulseaudio_srcs",
+    out_shared_libs = [
+        "libpulse.so",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@libsndfile",
+        "@tdb",
+        "@check",
+    ],
+)
