--- /dev/null	2024-06-02 19:15:58
+++ BUILD	2024-06-02 19:15:45
@@ -0,0 +1,63 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "ffmpeg_srcs",
+    srcs = glob([
+        "**",
+    ]),
+)
+
+configure_make(
+    name = "libffmpeg",
+    build_args = [
+        "-j8",
+    ],
+    generate_args = [
+        "-GNinja",
+    ],
+    configure_options = [
+        "--disable-programs",
+        "--disable-doc",
+        "--disable-lzma",
+        "--disable-htmlpages",
+        "--disable-manpages",
+        "--disable-podpages",
+        "--disable-txtpages",
+        "--disable-sdl2",
+    ] + select({
+        "@platforms//os:linux": [
+            "--enable-libxcb",
+            "--enable-libxcb-shm",
+            "--enable-libxcb-xfixes",
+            "--enable-libxcb-shape",,
+            "--enable-libpulse",
+         ],
+        "//conditions:default": [],
+    }),
+    env = select({
+        "@platforms//os:macos": {
+            "AR": "",
+        },
+        "//conditions:default": {
+        },
+    }),
+    lib_source = ":ffmpeg_srcs",
+    out_static_libs = [
+        "libavcodec.a",
+        "libavdevice.a",
+        "libavfilter.a",
+        "libavformat.a",
+        "libavutil.a",
+        "libswresample.a",
+        "libswscale.a",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@bzip2//:bz2",
+        "@iconv//:libiconv",
+        "@zlib",
+    ] + select({
+        "@platforms//os:linux": ["@libxcb"],
+        "//conditions:default": [],
+    }),
+)
