diff --git a/mediapipe/framework/tool/mediapipe_graph.bzl b/mediapipe/framework/tool/mediapipe_graph.bzl
index 22b43656..12fdbfd0 100644
--- a/mediapipe/framework/tool/mediapipe_graph.bzl
+++ b/mediapipe/framework/tool/mediapipe_graph.bzl
@@ -23,7 +23,6 @@ load("//mediapipe/framework/tool:build_defs.bzl", "clean_dep")
 # buildifier: disable=out-of-order-load
 # buildifier: disable=same-origin-load
 load("//mediapipe/framework/deps:descriptor_set.bzl", "direct_descriptor_set", "transitive_descriptor_set")
-load("@org_tensorflow//tensorflow/lite/core/shims:cc_library_with_tflite.bzl", "cc_library_with_tflite")
 
 def mediapipe_binary_graph(name, graph = None, output_name = None, deps = [], testonly = False, **kwargs):
     """Converts a graph from text format to binary format."""
@@ -164,22 +163,7 @@ def mediapipe_simple_subgraph(
             **kwargs
         )
     else:
-        cc_library_with_tflite(
-            name = name,
-            srcs = [
-                name + "_linked.cc",
-                graph_base_name + ".inc",
-            ],
-            tflite_deps = tflite_deps,
-            deps = [
-                clean_dep("//mediapipe/framework:calculator_framework"),
-                clean_dep("//mediapipe/framework:subgraph"),
-            ] + deps,
-            alwayslink = 1,
-            visibility = visibility,
-            testonly = testonly,
-            **kwargs
-        )
+        fail("TFLite support is disabled")
 
 def mediapipe_reexport_library(
         name,
