--- src/models/kv_cache.cpp.prev	2024-07-07 15:21:03
+++ src/models/kv_cache.cpp	2024-07-07 15:32:22
@@ -258,7 +258,7 @@
     : model_{model},
       state_{state},
       layer_count_{model_.config_->model.decoder.num_hidden_layers},
-      shape_{state_.params_->BatchBeamSize(), model.config_->model.decoder.num_key_value_heads, 1500, model.config_->model.decoder.head_size} {
+      shape_{state_.params_->BatchBeamSize(), model.config_->model.decoder.num_key_value_heads, 577 + state_.params_->sequence_length, model.config_->model.decoder.head_size} {
   values_.reserve(layer_count_ * 2);
 
   for (int i = 0; i < layer_count_; ++i) {
