--- /dev/null	2024-05-27 13:29:49
+++ BUILD	2024-05-27 13:29:32
@@ -0,0 +1,28 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+filegroup(
+    name = "libxau_srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
+    name = "libxau",
+    lib_source = ":libxau_srcs",
+    deps = [
+        "@xorgproto",
+    ],
+    env = select({
+        "@platforms//os:macos": {
+            "AR": "",
+        },
+        "//conditions:default": {
+        },
+    }),
+    out_shared_libs = [
+        "libXau.so",
+    ],
+    out_static_libs = [
+        "libXau.a",
+    ],
+    visibility = ["//visibility:public"],
+)
